src/MPI.jl

```diff
@@ -32,32 +35,22 @@ function __init__()
         end
     end
 
-    global const mpitype_dict = Dict{DataType, Cint}(
-        # Older versions of OpenMPI (such as those used by default in
```

Comment:
Can you leave this comment in? It explains why we're using older MPI features instead of newer ones that should in principle be preferred.